pkgs/by-name/as/asfa/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  ...
+}: let
+  version = "0.10.0";
+in
+  rustPlatform.buildRustPackage {
+    pname = "asfa";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "obreitwi";
+      repo = "asfa";
+      rev = "v${version}";
+      hash = "sha256-MnhnwtZmPFhOuiqNiaxJnPu88JOdlpvyVy0YGphblBc=";
+    };
+
+    cargoHash = "sha256-/bRBP/NzcNOXl/nANeOYouUAo3NNbtbV9fxIJrNajYQ=";
+
+    buildInputs = [
+      pkgs.libssh2
+    ];
+
+    nativeBuildInputs = with pkgs; [
+      gawk
+      help2man
+    ];
+
+    OPENSSL_DIR = lib.attrsets.getOutput "dev" pkgs.openssl;
```

Comment:
By adding `pkg-config` to `nativeBuildInputs` and adding `openssl` to `buildInputs` the rust build can automatically discover the `openssl` package, making this part not needed.

(But if you really needed this, I'd recommend using `lib.getDev` and `lib.getLib` instead of `lib.attrsets.getOutput`)

Reply:
Thanks, that is indeed much more straight forward :+1: 

For some reason it did not work like that in my first attempt at packaging in the original repo using [crane](https://github.com/ipetkov/crane); which, in hindsight, might have been overkill.